content/blog/entries/cc-vocabulary-docs-updates-2/contents.lr

```diff
@@ -0,0 +1,87 @@
+- The first draft of the vocabulary site is now live! I'm expecting a whole bunch of changes still, but here it is if you want to have a sneak peek: [https://cc-vocab-draft.surge.sh](https://cc-vocab-draft.surge.sh)
+-  I've consumed the [Github API](https://docs.github.com/en/free-pro-team@latest/rest) to get live release history, forks and starrers count. I think it adds a really nice touch to the site in general.
+- I've used [surge.sh](https://surge.sh) to deploy the draft site. I believe it's a really simple tool to have your site deployed within seconds!
+
+<center>
+<img alt"Github commit gif" src="github.png"/><br>
+<small class="muted">My github contribution chart is filling up!</small>
+</center>
+
+### What I've learnt
+
+Some say it's hard to learn through virtual internships. Well, let me prove you wrong. Here are my leanings in the past few weeks:
+
+- It's surprising how subjective (& yet objective) designing really is.
+- VueJS is _fantastic_. Maybe I'm a VueJS fan now. Should I remain loyal to ReactJS? I don't know.
+- Making a site responsive isn't the _easiest_ of tasks, but it's certainly doable after a lot of stretching & compressing; lets say that.
+- "Code formatting is essential" would be an _understatement_ to make.
+- Monorepo's have their own pro's and con's. But in our case the con's were negligible, thankfully! 
+-  I'll be following up with some performance & accessibility testing this coming week, so let's see how that plays out!
+- A mentor plays a vital role in any project. My mentor `@dhruvkb` has been very supportive and has made sure I stick to my timeline!
+
+### Other community work tidbits
+
+I believe apart from the internship work that I'm engaged in, I should also help around with some community PR work. I've been told I'm always welcome to, which is great!
+
+- I got the opportunity to speak at a CCOS event alongwith fellow speakers `@dhruvkb` & `@dhruvi`. I had a blast talking to budding students from DSC-IIT Surat & DSC-RIT.
+- The dark mode (as promised) should be out before my next blog post.
+- Deployed the vocabulary storybook on [chromatic](chromatic.com) and compared & contrasted the pros & cons. Snapshot testing in the near future maybe? 
+- Completed the hacktoberfest challenge. 
+
+### Bonus content
+Not many of you may know this, but this site uses the [lektor](https://getlektor.com) CMS. I needed to have it installed on my system (windows 10 OS) to run the code in our site repository. 
+Lektor suggests running the following code in powershell as an installation step:
+```powershell
+(new-object net.webclient).DownloadString('https://www.getlektor.com/installer.py') | python
+```
+
+I just didn't think this is a very elegant way. Being an ardent [chocolatey.org](chocolatey.org) fan, I just had to have it up on there! Now the installation step for lektor is simply:
```

Comment:
awesome that you did this 🎉

Reply:
Maybe we should update our documentation in the README.md to include the installation for windows!  😄